src/main/java/cz/cvut/kbss/termit/model/assignment/TermOccurrence.java

```diff
@@ -4,10 +4,16 @@
 import cz.cvut.kbss.termit.util.Vocabulary;
 
 import java.net.URI;
+import java.util.Objects;
 
 @OWLClass(iri = Vocabulary.s_c_vyskyt_termu)
 public abstract class TermOccurrence extends TermAssignment {
 
+    /**
+     * Suffix used to identify term occurrence contexts (named graphs) in the repository.
+     */
+    public static final String CONTEXT_SUFFIX = "occurrences";
```

Comment:
Why not to put it in Constants?

Reply:
I don't see a reason for putting this highly specific value into global application constants.